### ISSUE TITLE
Can't update extensions

### ISSUE THREAD

**Issue by Zai000** [Jays2Kings/tachiyomiJ2K#1740]
### Steps to reproduce

1- I updated all the extensions.
2- waited and it still didn't update.
3- I see a message "could not install extension".

### Expected behavior

It should update 

### Actual behavior

It didn't update 

### Crash logs

[2024-01-22-release.log](https://github.com/Jays2Kings/tachiyomiJ2K/files/14012678/2024-01-22-release.log)

### TachiyomiJ2K version

1.7.4

### Android version

Android 13

### Device

Galaxy A53 5G

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.4](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by Ekidoxx** [comment]
In my case the problem was in both TachiJ2K and TachiSY, and only with few extensions, so I guess it was the extension's apk fault somehow.

Just uninstalled & reinstalled the extensions and they are working fine so far.

**Comment by Zai000** [comment]
> In my case the problem was in both TachiJ2K and TachiSY, and only with few extensions, so I guess it was the extension's apk fault somehow.
> 
> Just uninstalled & reinstalled the extensions and they are working fine so far.

I tried this method and it works 👍🏻, uninstall all my extensions and reinstall (it still shows that it didnt install) but I restart the app and boom its prompt  trust

**Comment by Ekidoxx** [comment]
Yeah, installing/updating extensions from custom repos is buggy...

After hitting install, it doesn't recognize if the extension was installed or not, so it either allows infinite installations or after few tries finally realises it is installed and displays plain "installed"...
So right now at least two relaunches are needed for successful adding of new extension

install → \*relaunch\* → trust → \*relaunch\* → use

**Comment by WaitingKy** [comment]
strangely it works fine on WSA, i tried it on 2 PCs. but i got this same error on 3 of my android devices

**Comment by d4rky-pl** [comment]
I had the same issue on Android, the only workaround so far is to uninstall the extension and then install it again. This is unfortunately also buggy as the list of extensions is not updating correctly after installing. So the process right now is:

- uninstall the extension
- install the extension again (it will seem like it didn't install but that's not true, the UI just didn't update)
- restart the app
- trust the extension

**Comment by AdamHavlicek** [comment]
Created PR for #1734 which resolves the issue with not showing up extension after installation

**Comment by MyNameIsHakem** [comment]
Can you tell me where did you download your extensions ? as I don't know where I can download new ones ?

**Comment by Zai000** [comment]
> Can you tell me where did you download your extensions ? as I don't know where I can download new ones ?

From this link https://keiyoushi.github.io/extensions/
You can add the repo, or just download it manaully
